### PR TITLE
cloud: Determine cluster size

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by joey-yez — COMMENTED** [review]
1. we need links  for tpcc and sysbench 
https://www.tpc.org/tpcc/ 
https://github.com/akopytov/sysbench
2. remove IO(MBps) column from performance data.

File: tidb-cloud/size-your-cluster.md [hunk 1]
```diff
@@ -95,4 +99,223 @@ Minimum number of TiFlash<sup>beta</sup> nodes: `min((800 GB * 2 + 100 GB * 1) /
...
+    | Update Index      | 300     | 9,415  | 9,415  | 32            |
+    | Update Non-index  | 1,200   | 31,092 | 31,092 | 39            |
+
+- Maximum TPS and QPS
+
+    TPC-C performance:
+
+    | Transaction model | Threads | tpmC   | QPS    | Latency (ms) |
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 33,394 | 30,188 | 1,048.00     |
```
Comment (joey-yez) [inline tidb-cloud/size-your-cluster.md:186]: 不需要 “.00"

File: tidb-cloud/size-your-cluster.md [hunk 2]
```diff
@@ -95,4 +99,223 @@ Minimum number of TiFlash<sup>beta</sup> nodes: `min((800 GB * 2 + 100 GB * 1) /
...
+    | Update Index      | 150     | 4,346  | 4,346  | 35            |
+    | Update Non-index  | 600     | 13,603 | 13,603 | 44            |
+
+- Maximum TPS and QPS
+
+    TPC-C performance:
+
+    | Transaction model | Threads | tpmC   | QPS    | Latency (ms) |
+    |-------------------|---------|--------|--------|--------------|
+    | TPCC              | 1,200   | 15,208 | 13,748 | 2,321.00     |
```
Comment (joey-yez) [inline tidb-cloud/size-your-cluster.md:145]: 不需要 .00

**Review by joey-yez — COMMENTED** [review]
remove ".00" in data

**Review by joey-yez — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**